doc/source/installation.rst

```diff
@@ -33,9 +31,10 @@ Finally, the package is installed by running::
 Optimizing GPU acceleration
 ===========================
 
-To improve performance with an optimized CUDA implementation, you may install optional CUDA packages by running::
+To improve performance on PyTorch with an optimized CUDA implementation, you may install optional CUDA packages by
+running::
 
-    pip install -r requirements_optional_cuda.txt
+    pip install scikit-cuda cupy
```

Comment:
Why can't we have this in an *optional* requirements file again? Hardcoding dependencies is too brittle.

Reply:
@edouardoyallon

Reply:
@edouardoyallon

Reply:
@eickenberg is the one who knows.